.github/workflows/docker.yml

```diff
@@ -0,0 +1,94 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+
+env:
+  IMAGE_NAME: hello-world
```

Comment:
```suggestion
  IMAGE_NAME: vvgo-github-testing
```

Reply:
Ended up renaming to `vvgo`, since there's no point in having the extra `-github-testing`, especially since we can deploy the image from `docker.pkg.github.com` straight to Fly, once we've completed #15's Deploy Automation phase.

Reply:
Oh I misunderstood. I though this was for running test builds.